### PR TITLE
Python telemetry b2 release notes

### PR DESCRIPTION
**Pull request by mrm9084**

### CODE REVIEW

File: releaseNotes/PythonFeatureManagement.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 [Source code][source_code] | [Samples][samples]
 
+## 2.0.0b2 - October 14, 2024
+
+### Features
+
+* Added VariantAllocationPercentage, DefaultWhenEnabled to telemetry data when using `publish_telemetry`.
```
Comment (rossgrambo) [inline releaseNotes/PythonFeatureManagement.md:9]:
```suggestion
* Added VariantAllocationPercentage and DefaultWhenEnabled to telemetry data when using `publish_telemetry`.
```

File: releaseNotes/PythonFeatureManagement.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 [Source code][source_code] | [Samples][samples]
 
+## 2.0.0b2 - October 14, 2024
+
+### Features
+
+* Added VariantAllocationPercentage and DefaultWhenEnabled to telemetry data when using `publish_telemetry`.
```
Comment (rossgrambo) [inline releaseNotes/PythonFeatureManagement.md]:
```suggestion
* Added VariantAssignmentPercentage and DefaultWhenEnabled to telemetry data when using `publish_telemetry`.
```

**Review by rossgrambo — APPROVED** [review]

File: releaseNotes/PythonFeatureManagement.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 [Source code][source_code] | [Samples][samples]
 
+## 2.0.0b2 - October 14, 2024
+
+### Features
```
Comment (zhenlan) [inline releaseNotes/PythonFeatureManagement.md:7]: Should it be "Enhancement" to be consistent?

File: releaseNotes/AzureAppConfigurationProviderPython.md [hunk 4]
```diff
@@ -2,6 +2,16 @@
 
 [Source code][source_code] | [Package (Pypi)][package] | [Samples][samples]
 
+## 2.0.0b2 - October 14, 2024
+
+### Features
```
Comment (zhenlan) [inline releaseNotes/AzureAppConfigurationProviderPython.md:7]: Enhancement

File: releaseNotes/AzureAppConfigurationProviderPython.md [hunk 5]
```diff
@@ -2,6 +2,16 @@
...
+
+* Added AllocationId to Feature Flags when they have telemetry enabled.
```
Comment (zhenlan) [inline releaseNotes/AzureAppConfigurationProviderPython.md:9]: The way we put it made it sound like it's a new API. Suggest something like this:

Added support for including `AllocationId` to feature flag telemetry metadata when telemetry is enabled.

File: releaseNotes/PythonFeatureManagement.md [hunk 6]
```diff
@@ -2,6 +2,12 @@
...
+
+* Added VariantAssignmentPercentage, DefaultWhenEnabled to telemetry data when using `publish_telemetry`.
```
Comment (zhenlan) [inline releaseNotes/PythonFeatureManagement.md:9]: Not sure how it's related to `publish_telemetry`. Shouldn't these be included only when telemetry is enabled?

Comment (zhenlan) [reply releaseNotes/PythonFeatureManagement.md:9]: The `publish_telemetry` is what we ship for Azure Monitor, right? Does this sound reasonable?

Added support for including `VariantAssignmentPercentage` and `DefaultWhenEnabled` properties to telemetry when telemetry is enabled and published to Azure Monitor.

**Pull request merged**